### ISSUE TITLE
frand always returns 0.0

### ISSUE THREAD

**Issue by tanabi** [fuzzball-muck/fuzzball#411]
```
: main frand intostr tell ;
```

This will always return 0.  This is because the C function _int_f_rand is poorly written.

The code should be:

```
    return ((double)rand() / (double)RAND_MAX);
```

I will fix this as part of my documentation commit, which is something I wouldn't normally do but this is a real easy one.  So no need to do anything but close this ticket after I'm done documenting fbmath.c

**Comment by tanabi** [comment]
This is fixed by PR #412

**Comment by nyanster** [comment]
> What's funny about this one is I don't think this EVER worked, which tends to indicate nobody ever used frand (not even the person who implemented it). Or they used it once, found it was broken, and never reported it.

Agreed, I found it humorous (and unfortunate >w<) too. :B
